docs/TEAM.md

```diff
@@ -1,6 +1,6 @@
 ## Beginner
 | Image | Name       | Description |
-| ---------- | ---------- | ----------- |
+| <img src="https://avatars0.githubusercontent.com/u/70926567?s=460&u=7ff3dbfe0a665f95f12fcbdd53528debf76c2080&v=4" width="80px" height="80px">  | [Chioma Vera Nkanmuo](https://github.com/chiomavera)| Hi,I'm Chioma Vera Nkanmuo, Beginner Web developer from Nigeria with basic knowledge of  HTML and  CSS. I am currently in my penultimate level in university. Learning while contributing in making twindle a success.  |
```

Comment:
Also, instead of using such a long HTML markup for the image:

```
<img src="https://avatars0.githubusercontent.com/u/70926567?s=460&u=7ff3dbfe0a665f95f12fcbdd53528debf76c2080&v=4" 
	width="80px" height="80px">
```

You can simply use:

```
<img src="https://github.com/chiomavera.png?size=80">
```

This will look the same as above markup:

<img src="https://github.com/chiomavera.png?size=80">

This is based on [this Tip here](https://github.com/twindle-co/twindle/issues/492). Hope it helps.

Reply:
> Also, instead of using such a long HTML markup for the image:
> 
> ```
> <img src="https://avatars0.githubusercontent.com/u/70926567?s=460&u=7ff3dbfe0a665f95f12fcbdd53528debf76c2080&v=4" 
> 	width="80px" height="80px">
> ```
> 
> You can simply use:
> 
> ```
> <img src="https://github.com/chiomavera.png?size=80">
> ```
> 
> This will look the same as above markup:
> 
> ![](https://github.com/chiomavera.png?size=80)
> 
> This is based on [this Tip here](https://github.com/twindle-co/twindle/issues/492). Hope it helps.

OK, thanks